backend/src/main/java/io/tribehouse/motomo/user/User.java

```diff
@@ -0,0 +1,47 @@
+package io.tribehouse.motomo.user;
+
+import lombok.Builder;
+import lombok.Data;
+import org.hibernate.annotations.NaturalId;
+import org.hibernate.validator.constraints.Length;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Data
+@Table(name = "users")
+public class User {
```

Comment:
we need to keep layers more separated. 
we can't use the same model ex. `User` in controller, service and repository. 
For ex:
controller - UserRequest, UserResponse
service - User
repository - UserEntity

Reply:
Separeted User class to few classes[UserRequest - for Controllers, UserDto - for Services, UserEntity - for repository, UserResponse - for Controllers].